### PR TITLE
Modified JSON-DICT Implementation from TEXT to Text

### PR DESCRIPTION
**Pull request by Deepakdubey90**

### CODE REVIEW

**Comment by Deepakdubey90** [comment]
@minrk , Implemented the JSON field Implemented  as large-binary not Text becacuse oracle does not having data type as Text.

File: jupyterhub/orm.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ class JSONDict(TypeDecorator):
 
     """
 
-    impl = TEXT
+    impl = LargeBinary
```
Comment (minrk) [inline jupyterhub/orm.py]: Does it work to use `Text` instead of `TEXT`?

Comment (minrk) [reply jupyterhub/orm.py]: I understand that `Text` resolves to TEXT most places and CLOB on oracle.

Comment (minrk) [reply jupyterhub/orm.py]: Have you been able to test this with `Text`?

**Comment by Deepakdubey90** [comment]
Yes

**Comment by minrk** [comment]
Great! Are you planning to update the pull request? It is still using LargeBinary currently.

**Comment by minrk** [comment]
Went ahead and made the LargeBinary -> Text substitution. Thanks for opening this @Deepakdubey90 !

**Pull request merged**